scripts/run_validation.py

```diff
@@ -61,6 +61,8 @@ def validate_single_rule(cache, path, datasets, args, rule: dict = None):
         for dataset in datasets
     ]
     results = list(itertools.chain(*results))
+    if args.verbose_output:
+        print(f"{rule['core_id']} validation complete")
```

Comment:
Shouldn't it be a logger.info call?

Reply:
I will update. I'll also bump the version to version 0.5 for the release today